shiny/ui/_input_update.py

```diff
@@ -628,8 +628,8 @@ def update_select(
         An input label.
     choices
         Either a list of choices or a dictionary mapping choice values to labels. Note
-        that if a dictionary is provided, the keys are used as the (input) values so
-        that the dictionary values can hold HTML labels. A dictionary of dictionaries is
+        that if a dictionary is provided, the keys are used as the (input) values.It is not recommended to use
+        anything other than a string for these labels. A dictionary of dictionaries is
```

Comment:
Also make an update to `update_selectize()`

```suggestion
        that if a dictionary is provided, the keys are used as the (input) values. A dictionary of dictionaries is
```